cmd/network/show.go

```diff
@@ -91,6 +91,17 @@ var showCmd = &cobra.Command{
 			if node, err := registryConn.GetNode(ctx, idQuery); err == nil {
 				err = prettyPrint(node)
 				cobra.CheckErr(err)
+
+				if nodeStatus, err := registryConn.GetNodeStatus(ctx, idQuery); err == nil {
+					err = prettyPrint(nodeStatus)
+					cobra.CheckErr(err)
+				}
+				return
+			}
+
+			if nodeStatus, err := registryConn.GetNodeStatus(ctx, idQuery); err == nil {
```

Comment:
Why not always calling GetNodeStatus() above before the GetNode() block and without returning.